taskcluster/ci/release-notify-testrail/kind.yml

```diff
@@ -0,0 +1,71 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+task-defaults:
+    description: Sends Slack message to release testers that Testrail Milestone was created.
+    worker-type: b-android
+    worker:
+        docker-image: {in-tree: ui-tests}
+        max-run-time: 120
+        env:
+            TESTRAIL_PROJECT_ID: '59' # Fenix Browser
+            TESTRAIL_TEST_SUITE_ID: '49319' # Test Automation Release Milestone
```

Comment:
I created a new test suite in their project to use just for this called `Test Automation Release Milestone`. Based on our discussion from earlier, I'd like to use something more dynamic so all the test cases are updated automatically and accurate based on what tests are currently disabled/enabled at the time of the test run.

Reply:
If you are not modifying things, you should use the [Full Functional test suite](https://testrail.stage.mozaws.net/index.php?/suites/view/3192&group_by=cases:section_id&group_order=asc&group_id=385792) they use as the source of truth. That's the one where they add / remove / udpate manual tests and their references to automated tests

Reply:
i'd have to extend the functionality of `testrail_api.py` for that, which I can do. For that case, I'd need to add a method to `filter_request` or something to add the filters to return only test cases that have stuff like `Automation Status: Completed, Sub-Type: Smoke&Sanity` or whatever we want. 

I'll create a follow-up for this to add that because it'll definitely come up again, especially regarding auto-updating the test cases for each run based on the data-alignment work that's been done.

Reply:
As a workaround for now, I created the stand-alone test suite to use until I explored filtering with the TestRail API.

Does that work as a temporary solution and I can include that in Phase 4 or 5?

Reply:
That sounds perfect to me! Thanks!